prompting/utils/config.py

```diff
@@ -213,6 +219,13 @@ def add_miner_args(cls, parser):
         help="Set miner to stop on forward exception.",
     )
 
+    parser.add_argument(
+        "--neuron.should_force_model_loading",
+        type=bool,
+        default=False,
+        help="Force model loading independent of mock flag.",
```

Comment:
why?

Reply:
So we can use the mock flag to mock the subnetwork while being able to use a real model, this is specially used to accelerate benchmark development.